### PR TITLE
Add projector.project_preview and tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/actions/projector/project_preview.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                if id_ != projection_id
+            ]
+            instance["id"] = projector_id
+            instance["current_projection_ids"] = new_current_projection_ids
+            instance["preview_projection_ids"] = new_preview_projection_ids
+            instance["history_projection_ids"] = new_history_projection_ids
+            yield instance
+
+    def is_stable(self, value: Dict[str, Any]) -> bool:
+        return bool(value.get("stable"))
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/project_preview.py:87]:
```suggestion
        return value.get("stable", False)
```

File: tests/system/action/projector/test_project_preview.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                    "preview_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 99,
+                },
+                "projection/4": {
+                    "preview_projector_id": 3,
+                    "meeting_id": 1,
+                    "weight": 100,
+                },
+                "projection/5": {
```
Comment (FinnStutzenstein) [inline tests/system/action/projector/test_project_preview.py:44]: Please switch the order, so the projections are ordered by the projector. This makes it easier for a reader of the tests.

**Review by FinnStutzenstein — APPROVED** [review]
Looks good!

File: openslides_backend/action/actions/projector/project_preview.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/project_preview.py]: Okay. Changed.

File: tests/system/action/projector/test_project_preview.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/projector/test_project_preview.py:44]: Order of the projections changed. So it is easier to read.

File: openslides_backend/action/actions/projector/project_preview.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    def get_updated_instances(self, payload: ActionData) -> ActionData:
+        for instance in payload:
+            projection_id = instance.pop("id")
+            projection = self.datastore.get(
+                FullQualifiedId(Collection("projection"), projection_id),
+                ["preview_projector_id"],
+            )
+            # check if projection is from a preview projector
+            if not projection.get("preview_projector_id"):
+                continue
```
Comment (jsangmeister) [inline openslides_backend/action/actions/projector/project_preview.py:32]: Do we want to silently ignore if the client sends invalid data @FinnStutzenstein? I would suggest to throw an error here.

**Review by jsangmeister — APPROVED** [review]
Rest looks good from a programming point of view. Did not check the logic

File: openslides_backend/action/actions/projector/project_preview.py (hunk 3 above)
Comment (FinnStutzenstein) [reply openslides_backend/action/actions/projector/project_preview.py:32]: Yes, an error would be good.

Comment (reiterl) [reply openslides_backend/action/actions/projector/project_preview.py]: Throw an `ActionException` if the `preview_projector_id` is missing.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**